corehq/apps/enterprise/forms.py

```diff
@@ -30,14 +32,88 @@ class EnterpriseSettingsForm(forms.Form):
         widget=forms.Textarea(attrs={'rows': 2, 'maxlength': 512}),
     )
 
+    forms_filetype = forms.ChoiceField(
+        label=ugettext_lazy("Default File Type"),
+        required=False,
+        initial=ExportFileType.EXCEL_2007_PLUS,
+        choices=ExportFileType.CHOICES,
+    )
+
+    forms_auto_convert = forms.BooleanField(
+        label=ugettext_lazy("Automatically convert dates and multimedia links for Excel"),
+        required=False,
+        help_text=ugettext_lazy("Leaving this checked will ensure dates appear in excel format. "
```

Comment:
Would it be worth the effort to share this content between this file and `customize_export_new.html`?

It'd be cool to share even more of the form logic between the enterprise settings page and the export create/edit page, but I imagine it's a bigger undertaking involving using formsets in both places. I'm not even certain if the export create/edit page uses crispy.

Reply:
I like this idea, but think it should be addressed separately given that it will likely hold up this PR for a bit. I will add this to my todo list.